REST Bindings/openapi.yaml

```diff
@@ -152,11 +152,10 @@ paths:
               $ref: '#/components/headers/GS1-Extensions'
             Location:
               description: |
-                Absolute or relative URL of the capture job
+                Absolute or relative URL of the capture job.
               schema:
-                example: "/id9261379075"
+                example: "/capture/id9261379075"
```

Comment:
can we just show some example that does not involve a leading slash?

Reply:
yes but then we would have simply `id9261379075` which is potentially a little confusing...